### ISSUE TITLE
[Documentation] Button "Edit on GitHub" can't be clicked

### ISSUE THREAD

**Issue by FredericMartinez** [mnapoli/bref#313]

**Comment by atrope** [comment]
@mnapoli  where is the css? this should fix the edit button being clickable
```
@media only screen and (max-width: 992px) and (min-width: 768px)  {
article h1{
    max-width: calc(100% - 130px);
}
}

@media only screen and (min-width: 768px)  {
article h1{
    max-width: calc(100% - 60px);
}
}



```

**Comment by mnapoli** [comment]
Here is the CSS:

https://github.com/mnapoli/bref/blob/2a2c9cb5b3c5ea59d075a2f15b5bbe1fe15e3ebf/website/template/styles.css#L96-L98

**Comment by atrope** [comment]
PR Submitted

**Comment by mnapoli** [comment]
Closed in #325
